src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -96,6 +97,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
         "Unable to authenticate with Code Studio",
         "Did you set a valid token with the <options=bold>api</> and <options=bold>write_repository</> scopes?",
         "Try running `glab auth login` to re-authenticate.",
+        "Alternatively,  pass the `--gitlab-token` option. Example: `acli cs:wizard --gitlab-token=<TOKEN>`",
```

Comment:
```suggestion
        "Alternatively, pass the <options=bold>--gitlab-token</> option.",
```

Reply:
PR updated as per suggestion.

Reply:
I don;t think we need that here. Reason being this error will only be triggered once user has both hostname and token.
So `validateEnvironment()` is triggered and then we check for hostname. If host name is not available in options or in local config, we throw exception here https://github.com/acquia/cli/blob/master/src/Command/CodeStudio/CodeStudioWizardCommand.php#L326

I guess we can add that info to pass the hostname at this place